.gitignore

```diff
@@ -25,3 +25,11 @@ doc/_build
 
 # Generated man page
 doc/aws_hostname.1
+
+# Annoying macOS files
+.DS_Store
+._*
+
+# IDE configuration files
+.vscode
+.atom
```

Comment:
Add a newline plz

Reply:
No problem, I had actually thought about that, but I didn't do it because I noticed the original `.gitignore` file didn't have a new line at the end so I figured I'd keep it the same.